hooks/environment

```diff
@@ -59,16 +71,15 @@ dumpEnvSecrets() {
 }
 
 
-env_before="$(env | sort)" # used by 
+env_before="$(env | sort)" # used by
 
 # If we are using a specific key we should download and evaluate it
 if [[ -n "${BUILDKITE_PLUGIN_CLUSTER_SECRETS_KEY:-}" ]]; then
     secret=$(downloadSecret "${BUILDKITE_PLUGIN_CLUSTER_SECRETS_KEY}")
     processSecrets "${secret}"
-else
```

Comment:
removing this else branch is a breaking change, see the PR description for some thoughts on it